components/QuestionNotificationsToggleExplanationModal.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Modal, ModalHeader, ModalBody, ModalFooter, Button } from 'reactstrap'
+
+const QuestionNotificationsToggleExplanationModal = props => (
+  <Modal isOpen={props.isOpen} toggle={props.toggle}>
+    <ModalHeader>Question notifications</ModalHeader>
+    <ModalBody>
+      <h5>For students</h5>
+      <p>
+        If you enable notificaitons, you&apos;ll get notifications when your
```

Comment:
"If you enable notificaitons" => notifications

Reply:
Sorry, what do you mean exactly? Do you mean "Notificaitons: you'll get notifications when your..." like this?

Reply:
oh sorry, i meant "If you enable notificaitons" should be changed to "If you enable notifications", the current notifications is a typo

Reply:
Oh, right! I can fix that later along with the wording stuff I mentioned below.

Reply:
Please double check!